services/libs/data-access-layer/src/members/others.ts

```diff
@@ -78,3 +78,7 @@ export async function updateMemberReach(
     },
   )
 }
+
+export async function setMemberUpdatedAt(qx: QueryExecutor, memberId: string): Promise<void> {
```

Comment:
can we instead call this `touchMemberUpdatedAt` since we're not giving a date as input and set to that value, but instead setting it to `now()`